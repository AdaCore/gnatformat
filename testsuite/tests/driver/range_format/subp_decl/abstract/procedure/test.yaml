description: AbstractSubpDecl formatting for line selection.
driver: gnatformat
args:
  [
    "test.ads",
    "--range-format",
    "-SL 29",
    "-EL 31",
    "--pipe",
  ]
