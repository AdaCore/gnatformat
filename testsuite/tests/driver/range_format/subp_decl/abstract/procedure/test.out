test.ads(<AbstractSubpDecl ["Take"] test.ads:29:5-30:51>) - 29:1-30:51
^
   procedure Take (Element : out Element_Type; From : in out Set) is abstract;$

